screens/account/SignInOrRegister.tsx

```diff
@@ -38,8 +38,8 @@ const SignInOrRegister = () => {
   const toast = useToast();
 
   const [formData, setData] = useState<SignInFormData>({
```

Comment:
Why were the default sign in values changed to this?

Reply:
yeah so the local storage doesn't work anymore, so I didn't want to have to type in my credentials every time I wanted to test something. I think @tylerhm is working on a fix for this but in the mean time, I've just been doing this

It's a tester account, so feel free to use it yourself for testing purposes, or change this to your creds if you want to test with your account